lib/ansible/modules/commands/expect.py

```diff
@@ -194,11 +196,11 @@ def main():
             # Prefer pexpect.run from pexpect>=4
             out, rc = pexpect.run(args, timeout=timeout, withexitstatus=True,
                                   events=events, cwd=chdir, echo=echo,
-                                  encoding='utf-8')
+                                  encoding='utf-8', codec_errors=codec_errors)
         except TypeError:
             # Use pexpect.runu in pexpect>=3.3,<4
             out, rc = pexpect.runu(args, timeout=timeout, withexitstatus=True,
-                                   events=events, cwd=chdir, echo=echo)
+                                   events=events, cwd=chdir, echo=echo, codec_errors=codec_errors)
```

Comment:
`pexpect<4` does not support `codec_errors`.  If you make it here, and `codec_errors` was explicitly provided, an error needs to be raised explaining that `codec_errors` isn't supported.  As is, it would end up giving an error that may be misleading.

Reply:
@sivel where to raise the error?